test/users.test.js

```diff
@@ -0,0 +1,68 @@
+const request = require('supertest'),
+  app = require('../app');
+
+describe('Health check', () => {
+  test('It should show the uptime', () =>
+    request(app)
+      .get('/health')
+      .then(response => {
+        expect(response.statusCode).toBe(200);
+      }));
+});
+
+describe('POST /users. Test successful sign up', () => {
+  test('It should response with code 201', () =>
+    request(app)
+      .post('/users')
+      .send({
+        name: 'ArquiMedes Galileo',
+        last_name: 'Alighieri',
+        email: 'arquimedes.ali@wolox.ar',
+        password: '123e123_rw'
+      })
+      .then(response => {
+        expect(response.statusCode).toBe(201);
```

Comment:
Remember to add `dictum.chai()` when you get the response. https://github.com/Wolox/dictum.js#chai